src/Slider/Slider.js

```diff
@@ -284,18 +284,42 @@ module.exports = kind(
 	* @private
 	*/
 	jumpWrapperComponents: [
-		{name: 'buttonLeft', kind: IconButton, backgroundOpacity: 'transparent', classes: 'moon-slider-button left', icon: 'arrowlargeleft', onSpotlightKeyDown: 'configureSpotlightHoldPulse', onSpotlightSelect: 'previous', ondown: 'downLeft', onholdpulse: 'holdLeft', ondragstart: 'preventDrag', defaultSpotlightDisappear: 'buttonRight'},
-		{name: 'slider', classes: 'moon-slider', spotlight: true, accessibilityLive: 'polite'},
```

Comment:
Hi @clinuz 
No, i think it is my mistake when I fix merge conflict.
Thanks :+1: